README.md

```diff
@@ -72,4 +72,7 @@ remuxer.Finalize(&motion_photo);
 *see samples/remux.cc for complete example code*
 
 ## Testing
-This library has a set of unit tests that verify demuxing and remuxing functionality against a set of golden images. These tests depend on [googletest](http://github.com/google/googletest) and can be run with bazel using `bazel test //tests/demuxer:tests` or `bazel test //tests/remuxer:tests`.
+This library has a set of unit tests that verify demuxing and remuxing functionality against a set of golden images. These tests depend on [googletest](http://github.com/google/googletest) and can be run with bazel using `bazel test //tests/(demuxer|remuxer):tests`.
+
+## Fuzzing
+This library is setup to be fuzz tested with [libfuzzer](https://llvm.org/docs/LibFuzzer.html). Fuzzing can be run using `bazel test --config=(asan|msan)-libfuzzer //fuzz:(demuxer|remuxer)_fuzz_test`
```

Comment:
Maybe change to `//fuzz:all`

Reply:
It seems based on how the rule is setup they need to be run individually. (Running `//fuzz:all` leads to the tests trying to access some file at the same time)